src/renate/updaters/learner.py

```diff
@@ -92,6 +94,23 @@ def __init__(
         self._val_memory_buffer: DataBuffer = InfiniteBuffer()
         self._create_metrics_collections(logged_metrics)
         self._post_init()
+        self.save_hyperparameters(
+            ignore=[
+                "model",
+                "loss_fn",
+                "components",
+                "train_transform",
+                "test_transform",
+                "buffer_transform",
+                "train_transform",
+                "train_target_transform",
+                "test_transform",
+                "test_target_transform",
+                "buffer_transform",
+                "buffer_target_transform",
+                "logged_metrics",
+            ]
+        )
 
     def _post_init(self) -> None:
```

Comment:
This has moved. Can you clarify?

Reply:
This refers to the `_post_init` method. We initially had this method, because when recreating the `Learner` from a new state, we wouldn't call it's constructor. My understanding is that we now *do* call the constructor (via Lightning's reloading mechanism) so the stuff in `_post_init` could be moved to the regular `__init__`. Am I missing something?

Reply:
If the reason for this function is the previous `__new__` call, then this can done away with. Should this cleanup be part of this PR or later?